ui/raidboss/data/07-dt/raid/r3s.ts

```diff
@@ -87,11 +89,13 @@ const triggerSet: TriggerSet<Data> = {
       name: {
         en: 'Barbarous Barrage Uptime Knockback',
         de: 'Brutalo-Bomben Uptime Rückstoß',
+        ja: 'ボンバリアンボム吹き飛ばし無効のタイミング',
         cn: '击退塔uptime打法击退提示时机调整功能',
       },
       comment: {
         en: 'Select towers to dodge with knockback immunity.',
         de: 'Wähle welche Türme mit Rückstoß-Immunität genommen werden.',
+        ja: 'ノックバック無効で避ける塔を選択してください',
```

Comment:
```suggestion
        ja: '吹き飛ばし無効で避ける塔を選択してください',
```

Dispite you are using 吹き飛ばし above, should be better to keep it consistence.

Reply:
> Dispite you are using 吹き飛ばし above, should be better to keep it consistent.

Agreed, thank you for catching this consistency issue. I'll implement your suggested change as is.

commit: 52e01788579e959a054d392ceb42c1c8d4b92482